Mathlib/MeasureTheory/Integral/SetIntegral.lean

```diff
@@ -21,7 +21,7 @@ and is zero otherwise.
 
 Since `∫ x in s, f x ∂μ` is a notation, one can rewrite or apply any theorem about `∫ x, f x ∂μ`
 directly. In this file we prove some theorems about dependence of `∫ x in s, f x ∂μ` on `s`, e.g.
-`integral_union`, `integral_empty`, `integral_univ`.
+`integral_union`, `integral_empty`, `setIntegral_univ`.
```

Comment:
Can you rename `integral_union` and `integral_empty` as well? A follow-up PR is fine.

Reply:
Well spotted! I hate those names too 😁